### ISSUE TITLE
2020 11 03 ZIP Code results are the same and only one page of results.mp4

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#189]
Screengrab/videos (insert file names here)
2020 11 03 ZIP Code results are the same and only one page of results.mp4

Link
boston.gov/my-neighborhood

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer> MacOS 10.15.7 > Chrome 86.0

Trusted Tester Test

Brief Description
While testing for neighborhood services, using the ZIP Code 02109 elicits only one address and ZIP has thousands of addresses with this ZIP Code.
[2020 11 03 ZIP Code results are the same and only one page of results.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5489315/2020.11.03.ZIP.Code.results.are.the.same.and.only.one.page.of.results.mp4.zip)

Steps to reproduce
1)	00:00, Paste the link into a new browser window, press return.
2)	00:05, In the address field, enter the ZIP Code 02109
3)	00:08, Page loads and there is a page of results based on only the ZIP Code.
a)	One address is 1 Creek Sq, Boston, 02019
b)	10 Addresses are: 1 Devonshire Place, 1001, Boston, 02109
4)	This is both a Functional bug and an Accessibility bug. If the field provides results based only on the ZIP Code, all addresses with a ZIP Code of 02109 should be displayed, which does not occur.
5)	This is an Accessibility Bug as all addresses at 1 Devonshire Place are the same and can cause confusion to someone with a visual impairment, causing a lack of focus. 

Desired Result
•	Able to add an address to the Neighborhood search and find the resources available to me.

Actual Result
•	Only able to receive two addresses based on ZIP Code 02109.
